modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/utils/utils.js

```diff
@@ -123,112 +50,174 @@ class utils {
 		if (url) {
 			const currentBrowserPath = this.getCurrentBrowserPathWithoutHash();
 
+			//const currentBrowserPath = globals.window.location.pathname + globals.window.location.search;
```

Comment:
This line will make tests fail, as will using the `getCurrentBrowserPathWithoutHash` function outside the class 🤷

Reply:
But I guess we shouldn't commit commented out code right?

Reply:
Yeah, I'll amend this and https://github.com/liferay-frontend/liferay-portal/pull/489#discussion_r521246099 if green, then.

The whole `globals` thing is also another smell to me... but it's one that's also tricky to get rid of, so let's take it step by step... 😧